src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/MultiSourcePackageFeed.cs

```diff
@@ -27,6 +29,8 @@ public sealed class MultiSourcePackageFeed : IPackageFeed
 
         public bool IsMultiSource => _sourceRepositories.Length > 1;
 
+        public bool IsOnlyLoadingFromNuGetOrg => !IsMultiSource && _sourceRepositories.First<SourceRepository>().ToString() == NUGET_ORG_SOURCE_NAME;
```

Comment:
Ignore cases for string comparison. `StringComparer.OrdinalIgnoreCase.Equals(x, y)`